app/auto_posts.go

```diff
@@ -93,7 +93,7 @@ func (cfg *AutoPostCreator) CreateRandomPostNested(parentId, rootId string) (*mo
 		Message:   postText,
 		FileIds:   fileIds}
 	rpost, err2 := cfg.client.CreatePost(post)
```

Comment:
actually `err2` shouldn't be `err2` should be `resp` because is the response from the server (which includes the `Error` attribute).